packages/vxrn/src/create.ts

```diff
@@ -49,7 +50,9 @@ const extensions = [
 ]
 
 export const create = async (options: StartOptions) => {
-  const { host = '127.0.0.1', root, nativePort = 8081, webPort } = options
+  const { host = '127.0.0.1', root, port = 8081 } = options
```

Comment:
Why don't we assign a random port to this port too?

Reply:
We can. But ideally default to 8081 and let user override like this

Reply:
Let's keep 8081 as default - to be consistent with Metro (and also React Native apps looks by default for this port).